data/setupRoles.js

```diff
@@ -0,0 +1,21 @@
+const {getAccounts} = require('./utils');
+
+const setupRoles = async (linnia) => {
+  const accounts = await getAccounts();
+  const { users } = await linnia.getContractInstances();
+  accounts.forEach(async (account,i) => {
+  	if(i>0 && i<41){
```

Comment:
why 41?

Reply:
This is from the original script - first 40 users get added then the rest are providers.